File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, string alp) {
    string answer = "";
    //
    for (auto& e : my_string)
    {
        if (e == alp[0]) answer += e - 32;
        else answer += e;
    }
    //
    return answer;
}

File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.java
import java.util.*;

class Solution {
    public String solution(String my_string, String alp) {
        String answer = "";
        for (char ch : my_string.toCharArray()) {
            if (alp.contains(String.valueOf(ch))) {
                answer += String.valueOf(ch).toUpperCase();
            } else {
                answer += String.valueOf(ch).charAt(0);
            }
        }
        return answer;
    }
}